### DISCUSSION TITLE
Not able to transfer songs from one lazer version to another

### DISCUSSION THREAD

**Discussion by UnknownCoke** [ppy/osu#23476] [Q&A]
### Type

Game behaviour

### Bug description

When i downloaded the new update, I had 2 versions of the game.  When I opened the new version without deleting the old version, I could still access my songs.  When i deleted the old version, I had no access. I tried to merge the both on my macbook, but it says my files are locked. I checked and they weren't. I have a request to transfer songs from my old version to the new version. This has happened when a new update comes out. This has happened twice and I don't want to spend hours downloading all my songs back.  I still have all my files from my lazer on a external hard drive.  For me to transfer it, i would need to download stable and inmort all the songs and it takes hours to complete the transfer process from stable to lazer.  Please implement something for this. It is a huge hassle. 
P.s. I'm using a macbook pro silicon

### Version

2023.511.0

### Logs

<img width="1680" alt="Screenshot 2023-05-11 at 9 50 09 AM" src="https://github.com/ppy/osu/assets/133232107/dfc3fc7a-8736-41fc-b885-9371bc762f8c">

**Comment by peppy** [comment]
I have no idea how you broke your isntall, but lazer *does* have the ability to target a data folder that already exists.

You'll have to provide more details if you're looking for help.
